flink-autoscaler-plugin-jdbc/src/main/java/org/apache/flink/autoscaler/jdbc/state/JdbcAutoScalerStateStore.java

```diff
@@ -189,6 +191,28 @@ public void removeParallelismOverrides(Context jobContext) {
         jdbcStateStore.removeSerializedState(getSerializeKey(jobContext), PARALLELISM_OVERRIDES);
     }
 
+    @Override
+    public void storeConfigOverrides(Context jobContext, Configuration configOverrides) {
+        jdbcStateStore.putSerializedState(
+                getSerializeKey(jobContext),
+                CONFIG_OVERRIDES,
+                serializeParallelismOverrides(configOverrides.toMap()));
```

Comment:
```suggestion
                serializeConfigOverrides(configOverrides.toMap()));
```

It may be a bug.

Reply:
It's not because the serialization is identical but it could become a bug, you are right. Thanks for spotting!